### PR TITLE
Adding/modifying content from Troopers / PTS

### PR DESCRIPTION
**Pull request by jt-dd**
Changes:
* Adding slides from Troopers
* Adding slides from PassTheSalt workshop
* Adding Notebook from PassTheSalt
* Updating reference page with new conferences

### CODE REVIEW

File: docs/references.md [hunk 1]
```diff
@@ -1,11 +1,48 @@
...
+
+Also we showed how the tool has been built and lessons we have learned from the process.
+
 ## 2024 - InsomniHack 2024 presentation
 ### [Standing on the Shoulders of Giant(Dog)s: A Kubernetes Attack Graph Model](https://www.insomnihack.ch/talks-2024/#BZ3UA9) 
 
-[Recording :fontawesome-brands-youtube:{ .youtube } ](#){ .md-button  .md-button--youtube } [Slides :fontawesome-solid-file-pdf:{ .pdf } ](files/insomnihack24/Kubehound - Insomni'Hack 2024 - slides.pdf){ .md-button } [Dashboard PoC :fontawesome-brands-python:{ .python } ](https://github.com/DataDog/KubeHound/tree/main/deployments/kubehound/scripts/dashboard-insomnihack24k){ .md-button }
+[Recording :fontawesome-brands-youtube:{ .youtube } ](https://www.youtube.com/watch?v=sy_ijtW6wmQ){ .md-button  .md-button--youtube } [Slides :fontawesome-solid-file-pdf:{ .pdf } ](files/insomnihack24/Kubehound - Insomni'Hack 2024 - slides.pdf){ .md-button } [Dashboard PoC :fontawesome-brands-python:{ .python } ](https://github.com/DataDog/KubeHound/tree/main/deployments/kubehound/scripts/dashboard-demp){ .md-button }
```
Comment (Minosity-VR) [inline docs/references.md:43]: `dashboard-demo` for the link instead of `demp`?

File: docs/references.md [hunk 2]
```diff
@@ -1,11 +1,48 @@
 # References
 
+## 2024 - Pass The Salt (PTS) Workshop
+### [KubeHound: Identifying attack paths in Kubernetes clusters at scale with no hustle](https://cfp.pass-the-salt.org/pts2024/talk/WA99YZ/) 
+
+[Slides :fontawesome-solid-file-pdf:{ .pdf } ](files/PassTheSalt24/Kubehound - PassTheSalt - slides.pdf){ .md-button } [Jupyter notebook :fontawesome-brands-python:{ .python } ](https://github.com/DataDog/KubeHound/tree/main/deployments/kubehound/notebook/KubehoundDSL_101.ipynb){ .md-button }
```
Comment (Minosity-VR) [inline docs/references.md:6]: Hmm idk but the formatting seems broken on my side (even on main, it is not specific to this PR but noted it):
image

**Review by Minosity-VR — APPROVED** [review]
lgtm with a few nits

File: docs/references.md (hunk 2 above)
Comment (jt-dd) [reply docs/references.md:6]: did you use the local command to render it ? `make local-wiki`

Comment (Minosity-VR) [reply docs/references.md:6]: Nope, I thought it was some markdown syntax I didn't know about that would be understood by the github website.
If it's expected to be read locally, testing the local wiki but should be good :D

**Review by Minosity-VR — APPROVED** [review]

**Pull request merged**